crates/mev-share-sse/src/types.rs

```diff
@@ -184,7 +184,7 @@ impl<'de> Deserialize<'de> for FunctionSelector {
             return Err(serde::de::Error::custom(format!(
                 "Expected 4 byte function selector: {}",
                 hex_str
-            )))
+            )));
```

Comment:
```suggestion
            )))
```

Reply:
lol my lint-on-save wouldn't let me undo that